README.md

```diff
@@ -1,226 +1,139 @@
 
-- Install [docker](https://docs.docker.com/)
-- Install docker [compose](https://docs.docker.com/compose/install/) >1.6.1
-- Clone this project
-  `git clone git@github.com:JeroenBoersma/docker-compose-development.git development`
+# How to get started
+Make sure that you have the prerequisites installed and running correctly before proceeding.
 
-## Additional installation steps for OSX
-
-- Install [Docker Sync](http://docker-sync.io/) and its dependencies.
-- You need to install coreutils using Homebrew only to have realpath command. :)
-  `brew install coreutils`
-
-## Persistent data container
-
-For running the database you need to create a new persistent data volume `docker volume create --name dockerdev-mysql-volume`
-
-If you already were using this repo before (or want a local directory), you can map the existing volume with:
-`docker volume create -o 'type=none' -o 'device='${PWD}'/mysql' -o 'o=bind' dockerdev-mysql-volume`
-
-## Preparation
-
-Tested under Linux. Also tested on OSX by a limited number of developers.
-For Windows, take a look at the docker beta (heard that good performances are met)
-
-Stop all other local Webservers running on port 80/443, stop all MySQL database servers on port 3306.
-
-Set-up your database credentials (`conf/mysql`) and Blackfire (`conf/blackfire`) profile in the conf directory.
-
-- conf/mysql (`MYSQL_ROOT_PASSWORD=something`)
-
-## Blackfire
+## 1). Configure your environment
+ 1. Clone this repository:
+```
+git clone git@github.com:JeroenBoersma/docker-compose-development.git development
+```
+ 2. Create a new persistent data volume with:
+```
+docker volume create --name dockerdev-mysql-volume
+```
+Or, if you already were using this repository before (or want a local directory), you can map the existing volume with:
+```
+docker volume create -o 'type=none' -o 'device='${PWD}'/mysql' -o 'o=bind' dockerdev-mysql-volume
+```
+ 3. Configure you MySQL credentials by editing [conf/mysql](conf/mysql). 
+ 4. Start your containers!
+ ```
+ ./bin/dev up
+ ```
+ Or, if you are on OSX:
+ ```
+ docker-sync start
+ ``` 
 
-Blackfire is optional, the library is already available in PHP, but not loaded by default.
-To load blackfire, simply add a `conf/blackfire` file with content from the [Blackfire website](https://blackfire.io/docs/integrations/docker#documentation).
-Copy/paste the contents from the first block in the configuration file and remove the `export ` from the beginning of the line.
-`./bin/dev/down` and `./bin/dev/up` to load blackfire.
+A optional, but recommended, step to take is to add the provided `.bin/dev` command to your system so you can use its commands anywhere you like.
 
-## Start
+ 1. Run `./bin/dev profile` from the development folder.
+ 2. Copy the output into `~/.bashrc` or `~/.bash_aliases`, on your own preference. 
+ 3. Log out and log back in for this to take effect. You could also just source the new files in your current terminal `. ~/.bashrc`.
```

Comment:
New terminal, how come I didn't think of that -.-'... 
Will write this down.

Also, what explanation are you looking for? Like, a example of the new registered `dev` ?